CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changes Since v2.6.0
+
+  * Singularity is now written primarily in Go to bring better integratiion with the existing container ecosystem
```

Comment:
dashes (`-`) instead of bullets (`*`) for consistency with previous readme style.

Reply:
Fair enough!